### DISCUSSION TITLE
ebus adapter v3 CAD

### DISCUSSION THREAD

**Discussion by wairlex** [john30/ebusd#577] [General]
Hello, is it possible to get ebus adapter v3 base board CAD file, in order to make a case?

Thanks!

**Comment by netweaver1970** [comment]
they don't want to share it (while the nice 3d model clearly exists), I already asked some time ago. 
There are box designs floating around already for it but I wanted also space for a methane gas detector module. So I recreated a skeleton of the board+ the Wemos D1 mini for my custom case. It was my 1st real Fusion360 parametric case, so I'm not proud of the workflow. But it did the job.

Anyway, You can use the skeleton (STEP compressed to ZIP) if you like. Make something nice!

[EBUSD adapter full.zip](https://github.com/john30/ebusd/files/8240853/EBUSD.adapter.full.zip)

**Reply by wairlex** [reply]
Thank you so much for sharing your step with us!

I was looking for the CAD to build a little box with a 3d printer, for not to have the adapter hanging from my boiler and by the way... learn and practice with Fusion360 (like you).
In my opnion, they could share the CAD with out worrying about the PCB being replicated.

**Comment by john30** [comment]
some guys already created cases, see here: https://github.com/eBUS/adapter/issues/9
